spec/webpack.spec.js

```diff
@@ -54,36 +65,50 @@ function runWebpack(callback, config) {
     let matches;
     let sriMatches;
 
-    while ((matches = cssMatcher.exec(html))) {
+    while ((matches = cssAssetMatcher.exec(html))) {
       cssAssets.push(matches[1]);
       cssCrossOrigin.push(/crossorigin="anonymous"/.test(matches[2]));
       if (sriMatches = /integrity="(sha[^"]+)"/.exec(matches[3])) {
         cssSri.push(sriMatches[1]);
       }
     }
-    while ((matches = cssMatcher.exec(html2))) {
+    while ((matches = cssAssetMatcher.exec(html2))) {
       cssAssets2.push(matches[1]);
       cssCrossOrigin2.push(/crossorigin="anonymous"/.test(matches[2]));
       if (sriMatches = /integrity="(sha[^"]+)"/.exec(matches[3])) {
         cssSri2.push(sriMatches[1]);
       }
     }
 
-    while ((matches = jsMatcher.exec(html))) {
+    while ((matches = jsAssetMatcher.exec(html))) {
       jsAssets.push(matches[1]);
       jsCrossOrigin.push(/crossorigin="anonymous"/.test(matches[2]));
       if (sriMatches = /integrity="(sha[^"]+)"/.exec(matches[3])) {
         jsSri.push(sriMatches[1]);
       }
     }
-    while ((matches = jsMatcher.exec(html2))) {
+    while ((matches = jsAssetMatcher.exec(html2))) {
       jsAssets2.push(matches[1]);
       jsCrossOrigin2.push(/crossorigin="anonymous"/.test(matches[2]));
       if (sriMatches = /integrity="(sha[^"]+)"/.exec(matches[3])) {
         jsSri2.push(sriMatches[1]);
       }
     }
 
+    while ((matches = cssPreloadMatcher.exec(html))) {
+      cssPreload.push(/rel="preload"/.test(matches[2]) && /as="style"/.test(matches[2]));
+    }
+    while ((matches = cssPreloadMatcher.exec(html2))) {
+      cssPreload2.push(/rel="preload"/.test(matches[2]) && /as="style"/.test(matches[2]));
+    }
```

Comment:
Help needed! These matches are empty since the manipulated content was not represented in the HTML yet.